server/middleware/error_handler.js

```diff
@@ -33,8 +33,11 @@ module.exports = (err, req, res, next) => {
     err.code = err.status || code;
   }
 
-  console.error('Error Express : ', err);
-  if (err.stack) console.log(err.stack);
+  // We don't polute the tests results unless with explicitly run them with the DEBUG env variable
+  if(process.env.DEBUG) {
```

Comment:
You are right, might be an issue.
I've changed it to 
    
    // We don't pollute the tests results when running mocha unless the DEBUG variable is set
    if(process.env.DEBUG || !process.argv[1].match(/\/mocha\//))